staging/src/k8s.io/api/go.mod

```diff
@@ -21,7 +21,7 @@ require (
 	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
 	github.com/modern-go/reflect2 v1.0.2 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
-	github.com/rogpeppe/go-internal v1.10.0 // indirect
+	github.com/rogpeppe/go-internal v1.11.0 // indirect
```

Comment:
didn't this already merge in https://github.com/kubernetes/kubernetes/pull/119086/commits/39a4cd1a083f0dff5ff3fc78063b64ad3c61099f

Reply:
Yes. After #119086 is merged, this PR can be closed.
/close